CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 3.2.1: 2020-11-18
+
```

Comment:
```suggestion
```

I'll fill in the release version and date after merge and an internal test run if that's ok.